app.R

```diff
@@ -2,7 +2,7 @@
 # To deploy, run: rsconnect::deployApp()
 # Or use the blue button on top of this file
 
-Sys.setenv(R_CONFIG_ACTIVE = "default") # Replace "default" with your config
+Sys.setenv(R_CONFIG_ACTIVE = "veoibd") # Replace "default" with your config
```

Comment:
I'd like to keep the main package on GH set to 'default' here. This can be changed to 'veoibd' on the server.